### DISCUSSION TITLE
Support to withoutRelations param

### DISCUSSION THREAD

**Discussion by trippo** [spatie/laravel-queueable-action#65] [Ideas]
I think is a good practice to use withoutRelations on job constructor like Laravel job, to prevent relations from being serialized.

[withoutRelations doc](https://laravel.com/docs/8.x/queues#handling-relationships)

Is it possible to include a param to use this functionality?

**Comment by brendt** [comment]
We're not adding any new features to this package. It's only intended to handle basic jobs, if you need more fine grained control, you should use a dedicated job instead.

**Reply by jaspermutsaerts-paqt** [reply]
I did not realize there was a discussions tab. (Finding my bearings contributing to open source, only checked Issues)

This makes it clear that my PR #113 won't be accepted. But maybe it's useful for @trippo or anyone else looking, for their own version.
